sunholmexp.py

```diff
@@ -284,15 +298,22 @@ def list_previous_update(n=0) -> None:
     print("\n".join(process_event(events[n-1], state)))
 
 
-def list_current_state(filter_player: str="") -> None: # todo this is not the right function but I am co-opting it for testing
+def list_current_state(filter_player: str="", sortby: str="") -> None: # todo this is not the right function but I am co-opting it for testing
     events = get_event_list()
 
     state = State()
 
     for event in events:
         process_event(event, state)
 
-    for player in state.players:
+    player_iter = state.players.keys()
+    if sortby == "exp":
+        player_iter = sorted(player_iter, key=lambda name: state.players[name])
+    elif sortby == "name":
+        player_iter = sorted(player_iter)
+    else:
```

Comment:
I have not tested this but the default for `sortby` here is `""`. Would that not automatically trigger this `else` instead of being ignored?

Reply:
Good catch, did not test that.